README.md

```diff
@@ -27,6 +26,7 @@ In this context, a `slug` is a string with a malleable, arbitrary value. For som
 
 ### Non-deterministic Platform Show Links
 * Amazon Music: `https://music.amazon.com/podcasts/${uniquePlatformID}`
+* Breaker: `https://breaker.audio/${slug}`
```

Comment:
```suggestion
* Breaker: `https://breaker.audio/${uniquePlatformID}`
```

For the purposes of this document, `slug` is defined as a string with a malleable, arbitrary value. `uniquePlatformID` is the correct term.